NitroxPatcher/Patches/Client/CyclopsMotorModeButton_OnClick_Patch.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Harmony;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.Helper;
+using NitroxModel.Helper.GameLogic;
+using System;
+using System.Reflection;
+
+namespace NitroxPatcher.Patches.Client
+{
+    public class CyclopsMotorModeButton_OnClick_Patch : NitroxPatch
+    {
+        public static readonly Type TARGET_CLASS = typeof(CyclopsMotorModeButton);
+        public static readonly MethodInfo TARGET_METHOD = TARGET_CLASS.GetMethod("OnClick", BindingFlags.Public | BindingFlags.Instance);
+
+        public static bool Prefix(CyclopsMotorModeButton __instance)
+        {
+            SubRoot cyclops = (SubRoot)__instance.ReflectionGet("subRoot");
+            if (cyclops != null)
+            {
+                CyclopsHelmHUDManager cyclops_HUD = cyclops.gameObject.RequireComponentInChildren<CyclopsHelmHUDManager>();
+                return (cyclops_HUD.hornObject.activeSelf && (bool)cyclops_HUD.ReflectionGet("hudActive"));
```

Comment:
Is it really necessary to check if the horn is active? Isn't `hudActive` enough?